src/test/java/runge/wordledictionary/WordleGameTest.java

```diff
@@ -0,0 +1,43 @@
+package runge.wordledictionary;
+
+import org.junit.jupiter.api.Test;
+
+import java.io.CharArrayReader;
+import java.io.IOException;
+import java.util.Arrays;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class WordleGameTest
+{
+    @Test
+    public void WordleGameCorrectTest() throws IOException
+    {
+
+        //given
+        WordleGame game1 = new WordleGame();
+
+        //when
+        String correctWord = game1.getRandomWord();
+        CharResult[] correctArray = {CharResult.Correct, CharResult.Correct, CharResult.Correct, CharResult.Correct, CharResult.Correct};
+        CharResult[] status = game1.guess(correctWord);
+
+        //then
+        assertArrayEquals(correctArray, status);
+
+    }
+
+    @Test
+    public void WordleGameIncorrectTest() throws IOException
+    {
+        //given
+        WordleGame game1 = new WordleGame();
+
+        //when
+        CharResult[] notCorrect = game1.guess("xxxxx");
+        CharResult[] correctArray = {CharResult.Correct, CharResult.Correct, CharResult.Correct, CharResult.Correct, CharResult.Correct};
+
+        //then
+        assertFalse(false, Arrays.toString(notCorrect));
```

Comment:
I don't know what this line is supposed to do.

Reply:
sets the boolean to false and will only pass the test if it returns false...right?

Reply:
if notCorrect is false aka it fails, then the test will return true

Reply:
`notCorrect` isn't a boolean.

Reply:
how do I check to see if the two arrays equal then?

Reply:
You expect the two arrays to be equal?

Reply:
no I know for sure they won't equal, since the word I'm using doesn't exist. but all tests have to pass so I want to create one that will only pass if the two arrays don't equal

Reply:
So how can you express this in code?

Reply:
can I make an if statement that creates a boolean statement and then check that to my false?